tox.ini

```diff
@@ -137,9 +137,7 @@ deps = bandit
 commands = bandit -r exec_helpers
 
 [testenv:dep-graph]
-envdir = {toxworkdir}/dep-graph
 deps =
-    pipenv
-commands =
-    pipenv install -r {toxinidir}/build_requirements.txt --skip-lock
-    pipenv graph
+    .
+    pipdeptree
```

Comment:
is it already present as requirement?

Reply:
it's special case: we install self not in edit mode (use proper install) and then install `pipdeptree` as utility to get dependency tree.

on call we see:
```
exec-helpers==1.3.0
  - advanced-descriptors [required: >=1.0, installed: 1.0.2]
    - six [required: >=1.9, installed: 1.11.0]
    - typing [required: >=3.6, installed: 3.6.4]
  - paramiko [required: >=2.4, installed: 2.4.1]
    - bcrypt [required: >=3.1.3, installed: 3.1.4]
      - cffi [required: >=1.1, installed: 1.11.5]
        - pycparser [required: Any, installed: 2.18]
      - six [required: >=1.4.1, installed: 1.11.0]
    - cryptography [required: >=1.5, installed: 2.2.2]
      - asn1crypto [required: >=0.21.0, installed: 0.24.0]
      - cffi [required: >=1.7, installed: 1.11.5]
        - pycparser [required: Any, installed: 2.18]
      - idna [required: >=2.1, installed: 2.6]
      - six [required: >=1.4.1, installed: 1.11.0]
    - pyasn1 [required: >=0.1.7, installed: 0.4.3]
    - pynacl [required: >=1.0.1, installed: 1.2.1]
      - cffi [required: >=1.4.1, installed: 1.11.5]
        - pycparser [required: Any, installed: 2.18]
      - six [required: Any, installed: 1.11.0]
  - PyYAML [required: >=3.12, installed: 3.12]
  - six [required: >=1.10.0, installed: 1.11.0]
  - tenacity [required: >=4.4.0, installed: 4.12.0]
    - six [required: >=1.9.0, installed: 1.11.0]
  - threaded [required: >=1.0, installed: 1.0.3]
    - six [required: >=1.9, installed: 1.11.0]
    - typing [required: >=3.6, installed: 3.6.4]
  - typing [required: >=3.6, installed: 3.6.4]
```